timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TestableTimelockCluster.java

```diff
@@ -195,7 +197,8 @@ public WaitForLocksResponse waitForLocks(WaitForLocksRequest request) {
 
     public StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransaction(
             StartIdentifiedAtlasDbTransactionRequest request) {
-        return timelockService().startIdentifiedAtlasDbTransaction(request);
+        return timelockService(defaultClient, request.requestorId())
+                .startIdentifiedAtlasDbTransaction(ImmutableIdentifiedTimeLockRequest.of(request.requestId()));
     }
```

Comment:
If within this class we expose the RPC client, the implementation can be a little simpler:

```
    private TimelockRpcClient rpcClient(String client) {
        return proxies.failoverForClient(client, TimelockRpcClient.class);
    }

    public StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransaction(
            StartIdentifiedAtlasDbTransactionRequest request) {
        return rpcClient(defaultClient).startIdentifiedAtlasDbTransaction(request);
    }
```

(and you won't need the timelockService(String, UUID) method nor the DefaultTimelockService factory that takes a UUID)